cabal-install/tests/IntegrationTests.hs

```diff
@@ -0,0 +1,276 @@
+-- | Groups black-box tests of cabal-install and configures them to test
```

Comment:
Nope, it just looked like `tasty-golden` does something similar. I haven't read the code closely.